### PR TITLE
Remove dependency on packaging Python package

### PR DESCRIPTION
**Pull request by jfckm**
Signed-off-by: Jan Musial <jan.musial@intel.com>

---
This change is [Reviewable](https://reviewable.io/reviews/open-cas/open-cas-linux/1033)

### CODE REVIEW

File: utils/casctl [hunk 1]
```diff
@@ -4,15 +4,16 @@
...
 import sys
 
-min_ver = version.parse("3.6")
-ver = version.parse(platform.python_version())
+min_ver = "3.6"
+ver = platform.python_version()
 if ver < min_ver:
-    print((f"Minimum required python version is {min_ver}\n"
-            f"Detected python version is {ver}"),
+    print("Minimum required python version is {}. Detected python version is {}".format(
```
Comment (robertbaldyga) [inline utils/casctl]: Is there any reason to replace f-string with `format()`?

**Review by robertbaldyga — CHANGES_REQUESTED** [review]

File: utils/casctl (hunk 1 above)
Comment (jfckm) [reply utils/casctl]: yup, f-strings were introduced in Python 3.6

**Review by mmichal10 — APPROVED** [review]

File: utils/casctl (hunk 1 above)
Comment (robertbaldyga) [reply utils/casctl]: and our min version, defined in the same file is also 3.6

Comment (robertbaldyga) [reply utils/casctl]: ooh, I see

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**